library/src/jvmTest/kotlin/com/google/fhir/SerializationRoundTripTest.kt

```diff
@@ -39,8 +39,6 @@ val exclusionListR4 =
 
     // Time with :00
     "MedicationRequest-medrx0335.json",
```

Comment:
To be resolved. The generated `MedicationRequest` uses `Time` primitive class that should be serialized. The value property (of type `LocalTime`) should use the custom `LocalTimeSerializer`.

Reply:
This is now solved by adding the following in the generated files:

```
@file:UseSerializers(
  DoubleSerializer::class,
  LocalTimeSerializer::class,
)
```

Thanks @ellykits this is great